unity-renderer/Assets/UIComponents/Scripts/Components/GridContainer/GridContainerComponentView.cs

```diff
@@ -370,11 +370,17 @@ public void SetItems(List<BaseComponentView> items)
         SetItemSize(model.itemSize);
     }
 
-    public void AddItem(BaseComponentView item)
+    public void AddItemWithResize(BaseComponentView item)
     {
         CreateItem(item, $"Item{instantiatedItems.Count}");
         SetItemSize(model.itemSize);
     }
+    
+    public void SetItemSizeForModel() =>
+        SetItemSize(model.itemSize);
+    
+    public void AddItem(BaseComponentView item) => 
+        CreateItem(item, $"Item{instantiatedItems.Count}");
```

Comment:
We should add the `SetItemSizeForModel` and `AddItem` methods into the `IGridContainerComponentView` interface, right? And also update it in the notion documentation.

Reply:
to my surprise, there are no clients of this interface at all in the project. Only one implementation, which is `GridContainerComponentView`. So I don't know if it makes sense to add it to the interface 🤔 We need refactoring, where we will replace usage of `GridContainerComponentView `with interface `IGridContainerComponentView `or we need to remove this interface.

Reply:
Oh, it's true... I remember that this view was directly referenced from a parent prefab. I guess we could refactor it in order to avoid this direct reference and to work with the interface, but maybe it should be for a separated implementation.